### DISCUSSION TITLE
Github student

### DISCUSSION THREAD

**Discussion by fredidi** [community/community#145994] [GitHub Education]
### Select Topic Area

Question

### Body

Are there any student courses from Github?

**Comment by mecodeatlas** [comment]
Thanks for posting in the GitHub Community, @fredidi!

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Profile category is solely related to conversations related to GitHub profile page! This question should be in the GitHub Education category. I've gone ahead and moved it for you. Good luck!
